pkg/controller/constraint/constraint_controller.go

```diff
@@ -511,6 +468,16 @@ func (r *ReconcileConstraint) getOrCreatePodStatus(ctx context.Context, constrai
 	return statusObj, nil
 }
 
+func isRegoOnlyCT(ct *v1beta1.ConstraintTemplate) bool {
+	if len(ct.Spec.Targets[0].Code) == 0 {
+		return true
+	}
+	if len(ct.Spec.Targets[0].Code) == 1 && ct.Spec.Targets[0].Code[0].Engine == "Rego" {
```

Comment:
We should perform an affirmative check for VAP CEL, not a check for whether Rego is the only language.